build-tools/scripts/RunTests.targets

```diff
@@ -110,6 +110,7 @@
     <MSBuild
         ContinueOnError="ErrorAndContinue"
         Projects="$(_TopDir)\tests\RunApkTests.targets"
+        Targets="$(RunApkTestsTarget)"
```

Comment:
This property doesn't exist and I don't understand why it's needed.  The `DefaultTargets` value in `RunApkTests.targets` is `RunApkTests`, so building that file w/o specifying any targets will implicitly build the `RunApkTests` target.

Reply:
It is needed to run the `RunPerformanceApkTests` target instead of default `RunApkTests` target when we need it.